Lib/test/_test_multiprocessing.py

```diff
@@ -567,6 +567,11 @@ def handler(*args):
 
         return p.exitcode
 
+    def test_interrupt(self):
+        if os.name != 'nt':
+            exitcode = self._kill_process(multiprocessing.Process.interrupt)
+            self.assertEqual(exitcode, 1)
```

Comment:
```suggestion
    @unittest.skipIf(os.name == 'nt', "POSIX only")
    def test_interrupt(self):
        exitcode = self._kill_process(multiprocessing.Process.interrupt)
        self.assertEqual(exitcode, 1)
```

Can we exit with a SIGINT value instead? or will it always be 1?

Reply:
I believe we can exit with `SIGINT` if we unset the signal handler in the child process
```python
from signal import *
signal(SIGINT, SIG_IGN)
```
I expect this will make it behave similarly to `SIGTERM` or `SIGKILL`.

Reply:
We should still use `self.assertEqual(exitcode, signal.SIGINT)` by default (at least for the test suite).

Reply:
I think it is a good idea for the test suite to actually check against exit code 1 which *is* the default behavior.

`interrupt()` triggers a `default_int_handler` in the child process which raises `KeyboardInterrupt` which, in turn, will probably be processed in this chunk setting exit code to 1

https://github.com/python/cpython/blob/main/Lib/multiprocessing/process.py#L323-L327

Achieving this behavior is a major motivation for the change.

Reply:
Oh so there is an explicit value being used here. Can you actually add a comment about this as well? so that we know that Python *is* explicitly setting the exit code to 1. TiA.

Reply:
I added a unit test with `SIGINT` handler unset, a comment in the above test about exit code 1 and some hints in the docs about how it works with exceptions and exit codes.